enclave/core/sgx/calls.c

```diff
@@ -505,6 +506,100 @@ oe_result_t oe_get_enclave_status()
     return __oe_enclave_status;
 }
 
+/*
+**==============================================================================
+**
+** _debug_write_exit_frame()
+**
+** If a debug enclave, write the exit frame information to host's
+** ecall_context so that the host can stitch the ocall stack.
+**
+** This function is intended to be called by oe_asm_exit (see below).
+** When called, the call stack would look like this:
+**
+**     enclave-function -> oe_ocall -> oe_asm_exit -> _debug_write_exit_frame
```

Comment:
Can we break this up across multiple lines for readability? It doesn't look like it lines up with the code below where `oe_asm_exit` is used instead of `oe_exit_enclave`?

```
enclave-function
  -> oe_ocall
    -> oe_exit_enclave (aliased as __morestack)
      -> _debug_write_exit_frame
        -> oe_asm_exit
```

Reply:
Great suggestion. I was struggling to express it cleanly in comments. Your version of the call stack looks very clean.

Reply:
I have also made two changes with readability in mind:
- _debug_write_exit_frame calls oe_asm_exit and it mirrors your stack above. 
- Renamed _debug_write_exit_frame to _exit_enclave. If some one clicks on the oe_exit_enclave frame while debugging, the source code line would shoud up as a call to _exit_enclave instead of _debug_write_exit_frame. This is more readable i feel.